src/scripts/create_dansk.py

```diff
@@ -0,0 +1,176 @@
+"""Create the DANSK-mini NER dataset and upload it to the HF Hub."""
+
+from dataclasses import dataclass
+
+import pandas as pd
+from datasets import Dataset, DatasetDict, Split, load_dataset
+from huggingface_hub import HfApi
+from requests import HTTPError
+
+
+@dataclass
+class NamedEntity:
+    start: int
+    end: int
+    label: str
+
+
+@dataclass
+class Token:
+    id: int
+    start: int
+    end: int
+
+
+def main():
+    """Create the DANSK-mini NER dataset and uploads it to the HF Hub."""
+
+    # Define dataset ID
+    repo_id = "chcaa/DANSK"
+
+    # Download the dataset
+    dataset = load_dataset(path=repo_id, token=True)
+    assert isinstance(dataset, DatasetDict)
+
+    # This mapping is taken from Tedeschi, Simone, et al. "WikiNEuRal: Combined neural
+    # and knowledge-based silver data creation for multilingual NER." Findings of the
+    # Association for Computational Linguistics: EMNLP 2021. 2021.
+    ontonotes_tag_to_conll_tag_mapping: dict[str, str] = {
+        "CARDINAL": "O",
+        "DATE": "O",
+        "EVENT": "MISC",
+        "FACILITY": "LOC",
+        "GPE": "LOC",
+        "LANGUAGE": "MISC",
+        "LAW": "O",
+        "LOCATION": "LOC",
+        "MONEY": "O",
+        "NORP": "MISC",
+        "ORDINAL": "O",
+        "ORGANIZATION": "ORG",
+        "PERCENT": "O",
+        "PERSON": "PER",
+        "PRODUCT": "MISC",
+        "QUANTITY": "O",
+        "TIME": "O",
+        "WORK OF ART": "MISC",
+    }
```

Comment:
Why is it that we map to the conll2003 format? It seems simplifying without adding any benefits (in fact, differing from the 'standard' might be good at measuring weaknesses with LLMs, which perform poorly on multilevel classification, especially if there are a lot of labels).

Reply:
Hmm a really good point actually. Initially it was to be able to compare across languages, but since it seems like we're moving away from that anyway, I don't see why we can't change this too. This is also the case for several other NER datasets in that case (Swedish, Norwegian and Faroese all use different tagging systems).

Reply:
Yea, I believe it is great to standardize e.g. when training a model but in our case keeping the original labels is probably better. Unsure if the prompts for NER are written specifically for conll2003?

Reply:
Also, I think the NER module needs to change to reflect this change, somehow. Currently the label scheme is hardcoded, and the label scheme is also used in the few shot prompts. So not sure how to deal with it

Reply:
I can't find a full overview of the prompt, but I would probably create the label section and then either hardcode it pr. dataset or add a description pr. label and then parse it in automatically. 

the label scheme should ideally not be hardcoded. Thought it might be ideal to commit this PR for now and just create an issue for this.

Reply:
Added issue on it now: https://github.com/ScandEval/ScandEval/issues/96